src/scss/layout/_layout.scss

```diff
@@ -107,35 +147,85 @@ figcaption {
 			padding: rem(1) rem(60);
 			padding-bottom: rem(62);
 
-			.wp-block-image ~ h2 {
-				margin-top: $img-size-medium;
+			// WP customized styles for the WeCount website in Desktop and iPad Pro views
+			.wp-block-video,
+			.is-type-video {
+				min-width: auto;
 			}
-		}
-	}
 
-	/*
- * Default WP Alignment Classes
- *****************************************************************************/
+			.alignleft {
+				float: left;
+				margin-right: rem(25);
+				margin-top: 0;
+			}
 
-	.aligncenter,
-	.alignleft,
-	.alignright {
-		display: block;
-		padding: 0;
-	}
+			.alignright {
+				float: right;
+				margin-left: rem(25);
+				margin-top: 0;
+			}
 
-	.aligncenter {
-		float: none;
-		margin: 0.5em auto 1em;
-	}
+			// Styling for media and text block with media on the left
+			.wp-block-media-text {
+				display: grid;
+
+				.wp-block-media-text__content {
+					margin-left: rem(25);
+				}
 
-	.alignright {
-		float: right;
-		margin: 0.5em 0 1em 1em;
+				img {
+					width: 100%;
+				}
+			}
+
+			// Styling for media and text block with media on the right
+			.has-media-on-the-right {
+				.wp-block-media-text__media {
+					grid-column: 2;
+					grid-row: 1;
+				}
+
+				.wp-block-media-text__content {
+					grid-column: 1;
+					grid-row: 1;
+					margin-left: 0;
+					margin-right: rem(25);
+				}
+			}
+
+			.wp-block-group {
+				display: grid;
+
+				p {
+					margin: 0 auto;
+				}
+			}
+
+			.is-type-video .wp-block-embed__wrapper iframe,
+			.wp-block-video video {
+				height: rem(326);
+				width: rem(580);
+			}
+
+			.alignleft,
+			.alignright {
+				.wp-block-embed__wrapper iframe {
+					height: 100%;
+					width: 100%;
+				}
+			}
+		}
 	}
+}
 
-	.alignleft {
-		float: left;
-		margin: 0.5em 1em 1em 0;
+@media screen and (min-width: rem(1366)) {
```

Comment:
What's the reason to define a new breakpoint at 1366px? The css in this section looks like a duplicate of line 204-208.

Reply:
The reason to define this breakpoint is to adjust the size of aligned left or aligned right embedded media on desktop screens. Lines 210-215 adjust the size of aligned left or aligned right embedded media for iPad Pro screens. Lines 204-208 is the sized of embedded media that is NOT aligned right or aligned left for desktop and iPad Pro screens.

Reply:
The reason to define this breakpoint is to adjust the size of aligned left or aligned right embedded media on desktop screens. Lines 210-215 adjust the size of aligned left or aligned right embedded media for iPad Pro screens. Lines 204-208 adjust the size of embedded media that is NOT aligned right or aligned left for desktop and iPad Pro screens.

Reply:
Thanks for the explanation. The larger media iframe does look better on larger screen. Can you add a code comment here. Please also include that picking `1366px` is the starting px for ipad pro landscape mode.